codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -73,6 +74,7 @@ digitise->digitize
 digitised->digitized
 digitises->digitizes
 digitising->digitizing
+doughnut->donut
```

Comment:
Add plural `doughnuts`?

Reply:
I've added these, thanks for your suggestions.